src/Azure.Functions.Cli/Common/TemplatesManager.cs

```diff
@@ -47,7 +47,11 @@ private static async Task<IEnumerable<Template>> GetTemplates()
             }
 
             var templates = JsonConvert.DeserializeObject<IEnumerable<Template>>(templatesJson);
-            templates = templates.Concat(await GetNodeV4TemplatesJson()).ToList();
+            var extensionBundleMajorVersion = (await extensionBundleManager.GetExtensionBundleDetails()).Version[0];
+            if (extensionBundleMajorVersion == '2' || extensionBundleMajorVersion == '3')
```

Comment:
Michael had this check in his static template PR. I want to make sure that it is included.